phosphor-auditlog/auditLogParser.cpp

```diff
@@ -117,7 +101,7 @@ void ALParser::fillAuditEntry(nlohmann::json& parsedEntry)
 /**
  * @brief Strips '"' from beginning and end of value field
  */
-inline std::string getValue(std::string fieldText)
+inline std::string_view getValue(std::string_view fieldText)
```

Comment:
string_view is non owning, but I think how you use this that shouldn't be a problem.